streaming/src/main/scala/org/apache/spark/streaming/receiver/Receiver.scala

```diff
@@ -266,14 +266,16 @@ abstract class Receiver[T](val storageLevel: StorageLevel) extends Serializable
 
   /** Attach Network Receiver executor to this receiver. */
   private[streaming] def attachExecutor(exec: ReceiverSupervisor) {
```

Comment:
attachExecutor -> attachSupervisor since `executor_` is renamed to `_supervisor`